### PR TITLE
fix a problem about equivalence ecache feature 

### PR DESCRIPTION
**Pull request by huone1**
Signed-off-by: huone1 <huwanxing@huawei.com>
fix issue #1592

### CODE REVIEW

**Review by wpeng102 — APPROVED** [review]
/lgtm
/approve

File: pkg/scheduler/plugins/predicates/predicates.go [hunk 1]
```diff
@@ -308,6 +308,10 @@ func (pp *predicatesPlugin) OnSessionOpen(ssn *framework.Session) {
 				if err != nil {
 					fit, err = predicateByStablefilter(task.Pod, nodeInfo)
 					pCache.UpdateCache(node.Name, task.Pod, fit)
+				} else {
+					if !fit {
+						err = fmt.Errorf("plugin equivalence cache predicates failed")
+					}
```
Comment (Thor-wl) [inline pkg/scheduler/plugins/predicates/predicates.go:314]: if not fit, should return?

Comment (huone1) [reply pkg/scheduler/plugins/predicates/predicates.go:314]: Returned in line 320.

Comment (Thor-wl) [reply pkg/scheduler/plugins/predicates/predicates.go:314]: OK

**Comment by jasonliu747** [comment]
@huone1 sorry for any inconvenience.
image

**Review by wpeng102 — APPROVED** [review]
/lgtm

**Review by shinytang6 — APPROVED** [review]

**Pull request merged**